public/js/edit-autocomplete.js

```diff
@@ -68,6 +69,21 @@ const editAutocomplete = {
       currentLiEl.querySelector("input").value = item.value;
       currentLiEl.style.display = "flex";
     } else {
+      // TODO: Validat limit
```

Comment:
was there an issue with this validation? would be great to include this as well.